src/Plugin/QueueWorker/HubspotCommerceQueue.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drupal\hubspot_commerce\Plugin\QueueWorker;
+
+use Drupal\hubspot_commerce\Services\HubspotCommerce;
+
+use Drupal\Core\Annotation\QueueWorker;
+use Drupal\Core\Logger\LoggerChannelFactoryInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Class HubspotCommerceQueue.
+ *
+ * @QueueWorker(
+ *   id = "hubspot_commerce_queue",
+ *   title = @Translation("Hubspot Commerce queue"),
+ *   cron = {"time" = 60}
+ * )
+ */
+class HubspotCommerceQueue extends QueueWorker implements ContainerFactoryPluginInterface {
+
+  /**
+   * The Hubspot Commerce service.
+   *
+   * @var \Drupal\hubspot_commerce\Services\HubspotCommerce
+   */
+  protected $service;
```

Comment:
**CHANGE REQUEST**

What is this service for? Let's rename it to `protected $syncTo` implementing `\Drupal\commerce_hubspot\Hubspot\SyncToServiceInterface`. It's not common to have a `Services` folder as it's too generic, so we put services under an appropriately named folder based on their type e.g. `EventSubscriber`. Either leave them at the root folder as there won't be too many services in this module, or put them under a `Hubspot` folder where we'll put services that communicate with the HubSpot APIs.